forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -182,6 +190,27 @@ interface DeviceAuthenticator {
         return Calendar.getInstance().time
     }
 
+    /** Validate custom claims
+    * @param  customClaims: A map of custom claims to be validated
+    * @return Boolean value indicating whether the custom claims are valid or not
+     */
+    fun validateCustomClaims(customClaims: Map<String, Any>): Boolean {
+        return customClaims.keys.intersect(registeredKeys).isEmpty()
+    }
+
+    companion object {
+        val registeredKeys = listOf(
+            JWTClaimNames.SUBJECT,
+            JWTClaimNames.EXPIRATION_TIME,
+            JWTClaimNames.ISSUED_AT,
+            JWTClaimNames.NOT_BEFORE,
+            JWTClaimNames.ISSUER,
+            CHALLENGE,
+            PLATFORM,
+            ANDROID_VERSION
```

Comment:
@witrisna   PLATFORM and ANDROID_VERSION are not used in signing... I think we ned to remove them from here